src/Build/Logging/BinaryLogger/ProjectImportsCollector.cs

```diff
@@ -166,6 +170,16 @@ private void AddFileCore(string filePath)
             AddFileData(filePath, content, null);
         }
 
+        private void InvokeFileIOErrorEvent(string filePath, string message)
+        {
+            BuildEventArgs args = new BuildMessageEventArgs(
+                ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword("ProjectImportsCollectorFileIOFail", filePath, message),
+                null,
```

Comment:
it's good to specify the argument name with a colon for readability (for example right now I don't know what the null is for, without opening the IDE). Is it sender? Maybe good to specify "ProjectImportsCollector" as a sender so we can filter these messages if needed.